config/cageConfiguration.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?>
+<cageConfiguration>
+	<cageStructure>
+		<height>200</height>
+		<radius>60</radius>
+	</cageStructure>
+	<muscleUnits>
```

Comment:
add a place holder for motor parameters, like max force, etc

Reply:
you mean a new item with parameters? like:
<parameters max_force=50 .... />

Reply:
yea, kinda. we'll probably adapt it, but good to have a placeholder